railgun/cmd/rootcmd/rootcmd.go

```diff
@@ -25,5 +25,5 @@ var rootCmd = &cobra.Command{
 
 // Execute is the entry point to the controller manager.
 func Execute(ctx context.Context) {
-	rootCmd.ExecuteContext(ctx)
+	cobra.CheckErr(rootCmd.ExecuteContext(ctx))
```

Comment:
CheckErr w/o any specific action ?

Reply:
Please look at the code for `CheckErr`, it will halt the program if there is an error.

Reply:
Talked with @ccfishk in Slack, we're all set